oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -1184,3 +1238,12 @@ impl Runtime {
             .len() as i64
     }
 }
+
+/// Computes a Sha256 digest of `bytes` and returns it in a hex encoded string.
```

Comment:
```suggestion
/// Computes a SHA-256 digest of `bytes` and returns it in a hex encoded string.
```

for consistency with some of your own other comments